src/MICore/CommandFactories/gdb.cs

```diff
@@ -194,5 +199,16 @@ public override Task EnableTargetAsyncOption()
             // that isn't actually supported by gdb. 
             await _debugger.CmdAsync("kill", ResultClass.None);
         }
+        public override async Task Signal(string sig)
+        {
+            string command = String.Format("-interpreter-exec console \"signal {0}\"", sig);
+            await _debugger.CmdAsync(command, ResultClass.running);
```

Comment:
I am assuming you didn't use ConsoleCmdAsync because you are calling this from run mode? My one concern is that we are going to have a tricky time understanding output from the signal command since we will not be able to tell what it is from. Maybe that is okay if there is no output.